modules/apps/frontend-theme/frontend-theme-styled/src/main/resources/META-INF/resources/_styled/css/_clay_custom.scss

```diff
@@ -0,0 +1,13 @@
+.hide {
```

Comment:
Just FYI, `_clay_custom.scss` is specifically reserved for extending Clay CSS. This will add CSS at the end of `clay.css`. Adding these here will cause these styles to be missing if anyone added `_clay_custom.scss` to their theme.

Reply:
😮 I see, thanks for the info, I'll look for a better place! do you know any?

Reply:
No idea, maybe that shared css module in this pr or any file you're keeping in styled?

Reply:
Since the idea is to try to remove all the CSS from Styled. I created a new layer copying `_clay_custom.scss` and calling it `_clay_extend.scss`.

It might be interesting exploring new naming like:
`_clay_variables.scss` => `_clay_before.scss`
`_clay_extend.scss` => `_clay_after.scss`

Reply:
Renaming existing files = breaking change, probably not a good idea to do that right now. Are we getting rid of `css/portal/_aui.scss`? Why can't it just stay in there? The `overlay` classes are from AUI. If `.hide` is too hard to get rid of, we can just add an alias to `.d-none` in Clay CSS for it. I would say it's common enough to justify adding it.

Reply:
that's a good idea but I prefer to wait until we have moved all the classes, there will be more than just those 2

Reply:
@edalgrin just send a pr to Clay CSS for the aliases as you run into them. It will probably be merged and released by the time this pr gets approved. You won't need to find a place to put `.hide` in Portal.

Reply:
ok!